src/core/charpak_mod.F90

```diff
@@ -1,6 +1,8 @@
 !> \file
 !! \brief Module for character string manipulation
 !!
+!! Original code from https://github.com/geoschem/geos-chem/blob/main/Headers/charpak_mod.F90
```

Comment:
This is good but can you use the permalink (includes reference to the specific commit).